.github/workflows/it.yml

```diff
@@ -0,0 +1,111 @@
+    - name: containerd runc and crictl
+      run: |
+        sudo wget https://github.com/kubernetes-sigs/cri-tools/releases/download/v1.17.0/crictl-v1.17.0-linux-amd64.tar.gz
+        sudo tar zxvf ./crictl-v1.17.0-linux-amd64.tar.gz -C /usr/local/bin
+        sudo wget https://github.com/containerd/containerd/releases/download/v1.4.3/containerd-1.4.3-linux-amd64.tar.gz
+        mkdir containerd
+        sudo tar -zxf ./containerd-1.4.3-linux-amd64.tar.gz -C ./containerd
+        sudo mv ./containerd/bin/* /usr/bin/
+        sudo wget https://github.com/opencontainers/runc/releases/download/v1.1.2/runc.amd64 -O /usr/bin/runc
+        sudo chmod +x /usr/bin/runc
+    - name: Set up ossutils
+      run: |
+        sudo wget https://gosspublic.alicdn.com/ossutil/1.7.13/ossutil64 -O /usr/bin/ossutil64
+        sudo chmod +x /usr/bin/ossutil64
+    - uses: actions/checkout@v3
+    - name: Cache cargo
+      uses: Swatinem/rust-cache@v1
+      with:
+        target-dir: |
+          ./target-fusedev
+          ./target-virtiofs
+        cache-on-failure: true
+        key: ${{ runner.os }}-cargo-${{ matrix.arch }}
+
+    - name: Build nydus-rs
+      run: |
+        declare -A rust_arch_map=( ["amd64"]="x86_64" ["arm64"]="aarch64")
+        arch=${rust_arch_map[${{ matrix.arch }}]}
+        cargo install --version 0.2.1 cross
+        rustup component add rustfmt clippy
+        make -e ARCH=$arch -e CARGO=cross static-release
+        #sudo mv target-fusedev/$arch-unknown-linux-musl/release/nydusd nydusd-fusedev
+        #sudo mv target-fusedev/$arch-unknown-linux-musl/release/nydus-cached .
+        #sudo mv target-fusedev/$arch-unknown-linux-musl/release/nydus-image .
+        #sudo mv target-fusedev/$arch-unknown-linux-musl/release/nydusctl .
+        #sudo mv target-virtiofs/$arch-unknown-linux-musl/release/nydusd nydusd-virtiofs
+        sudo cp -r misc/configs .
+        sudo chown -R $(id -un):$(id -gn) . ~/.cargo/
+        pwd
+        ls -lh target-virtiofs/$arch-unknown-linux-musl/release
+    - name: Set up anchor file
+      env:
+        OSS_AK_ID: ${{ secrets.OSS_TEST_AK_ID }}
+        OSS_AK_SEC: ${{ secrets.OSS_TEST_AK_SECRET }}
+      run: |
+        sudo mkdir -p /home/runner/nydus-test-workspace
+        sudo cat > /home/runner/work/image-service/image-service/contrib/nydus-test/anchor_conf.json << EOF
+        {
+            "workspace": "/home/runner/nydus-test-workspace",
+            "nydus_project": "/home/runner/work/image-service/image-service",
+            "nydus_runtime_conf": {
+                "profile": "release",
+                "log_level": "info"
+            },
+            "registry": {
+                "registry_url": "localhost:5000",
+                "registry_namespace": "",
+                "registry_auth": "Y2h1YW5sYW5nLmdjdzoxcWF6MndzeA=="
+            },
+            "oss": {
+                "endpoint": "oss-cn-beijing.aliyuncs.com",
```

Comment:
There are only `skopeo` and `stargzify` binaries stored in `framework/bin` now which are being committed within this PR. Could we also store them into the OSS bucket?  We can improve the test framework to make it try to download from a OSS bucket. Otherwise, changes to the binaries make nydus repo bigger and bigger.
Actually, I can take this task to improve the frame work to avoid store many binaries in the code repo.